Triphouse-project/index.html

```diff
@@ -192,6 +192,36 @@ <h2 class="offer__title">What do we offer</h2>
         </div>
       </div>
     </section>
+    <section class="homes">
```

Comment:
Make your code prettier. I am a lot of times said about indentation on code.

Reply:
fixed code